### PR TITLE
Feature/add feedback service

### PR DESCRIPTION
**Pull request by SolovyovDmitry**

### CODE REVIEW

File: FeedbackService/FeedbackService.Broker/FeedbackService.Broker.csproj [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (MariyaSin) [inline FeedbackService/FeedbackService.Broker/FeedbackService.Broker.csproj]: do you need broker here?

**Comment by VLiamin** [comment]
You should not send token to feedback/creat.
But you should send token to feedback/find.
Now you do not  send it on feedback/find
photo_2022-08-09_00-27-49

File: Gateway/Clients/FeedbackServiceClients/FeedbackControllerClient.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    {
+      _httpContextAccessor = httpContextAccessor;
+
+      _client = new HttpClient();
+      _client.DefaultRequestHeaders.Accept.Clear();
+      _client.DefaultRequestHeaders.Accept.Add(
+        new MediaTypeWithQualityHeaderValue("application/json"));
+    }
+
+    public async Task<FindResultResponse<FeedbackInfo>> FindAsync(FindFeedbacksFilter filter)
```
Comment (MariyaSin) [inline Gateway/Clients/FeedbackServiceClients/FeedbackControllerClient.cs]: it  should not be here

Comment (MariyaSin) [reply Gateway/Clients/FeedbackServiceClients/FeedbackControllerClient.cs]: we will use find endpoint directly to the FeedbackService

File: Gateway/Clients/FeedbackServiceClients/FeedbackControllerClient.cs [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        }
+
+        result = JsonConvert.DeserializeObject<FindResultResponse<FeedbackInfo>>(
+          await response.Content.ReadAsStringAsync());
+      }
+
+      return result;
+    }
+
+    public async Task<OperationResultResponse<Guid?>> CreateAsync(CreateFeedbackRequest request)
```
Comment (MariyaSin) [inline Gateway/Clients/FeedbackServiceClients/FeedbackControllerClient.cs:30]: do we need return Guid?
I think it enough to return status code

File: Gateway/Clients/FeedbackServiceClients/FeedbackControllerClient.cs [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    {
+      OperationResultResponse<Guid?> result = new();
+      string token = _httpContextAccessor.HttpContext.Request.Headers["token"];
+
+      using (HttpRequestMessage message = new HttpRequestMessage(HttpMethod.Post, $"https://feedback.dev.ltdo.xyz/feedback/create"))
+      {
+        message.Headers.Add("token", token);
```
Comment (MariyaSin) [inline Gateway/Clients/FeedbackServiceClients/FeedbackControllerClient.cs]: we do not need the token in FeddbackService, the token do not work in other cloud
add create endpoint of FeddbackService to  skippedEndpoints (see appsetting.json)

File: Gateway/Clients/FeedbackServiceClients/FeedbackControllerClient.cs [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+        message.Content = JsonContent.Create(request);
+
+        HttpResponseMessage response = await _client.SendAsync(message);
+        if (response.StatusCode == HttpStatusCode.BadRequest)
+        {
+          throw new BadRequestException(response.Content.ReadAsAsync<HttpError>().Result.Message);
```
Comment (MariyaSin) [inline Gateway/Clients/FeedbackServiceClients/FeedbackControllerClient.cs]: do we need throw exseption? can you return the status code?

File: Gateway/Clients/FeedbackServiceClients/FeedbackControllerClient.cs [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+        }
+
+        if (response.StatusCode == HttpStatusCode.Forbidden)
+        {
+          throw new ForbiddenException(response.Content.ReadAsAsync<HttpError>().Result.Message);
```
Comment (MariyaSin) [inline Gateway/Clients/FeedbackServiceClients/FeedbackControllerClient.cs]: same

File: Gateway/Clients/FeedbackServiceClients/Interfaces/IFeedbackControllerClient.cs [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+using LT.DigitalOffice.FeedbackService.Models.Dto.Requests.Filter;
+using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.Kernel.Responses;
+
+namespace LT.DigitalOffice.Gateway.Clients.FeedbackServiceClients.Interfaces
+{
+  [AutoInject]
+  public interface IFeedbackControllerClient
+  {
+    Task<FindResultResponse<FeedbackInfo>> FindAsync(FindFeedbacksFilter filter);
```
Comment (MariyaSin) [inline Gateway/Clients/FeedbackServiceClients/Interfaces/IFeedbackControllerClient.cs]: remove

**Review by MariyaSin — COMMENTED** [review]
Please remove FeedbackService from here
you have same in the Feddback repository

File: FeedbackService/FeedbackService.Models.Dto/Requests/Filter/FindFeedbacksFilter.cs [hunk 8]
```diff
@@ -0,0 +1,15 @@
+﻿using LT.DigitalOffice.FeedbackService.Models.Dto.Enums;
```
Comment (MariyaSin) [inline FeedbackService/FeedbackService.Models.Dto/Requests/Filter/FindFeedbacksFilter.cs]: remove the model
gateway do not work with find endpoint

File: FeedbackService/FeedbackService.Models.Dto/Enums/FeedbackStatusType.cs [hunk 9]
```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
```
Comment (MariyaSin) [inline FeedbackService/FeedbackService.Models.Dto/Enums/FeedbackStatusType.cs]: remove the model

**Review by MariyaSin — APPROVED** [review]

**Review by VLiamin — APPROVED** [review]

**Pull request merged**